src/main/java/frc/team2412/robot/util/auto/AutoLogic.java

```diff
@@ -417,7 +412,7 @@ public static Command subwooferLaunch() {
 	}
 
 	public static Command visionLaunch() {
-		return (LAUNCHER_ENABLED && INTAKE_ENABLED && APRILTAGS_ENABLED
+		return (LAUNCHER_ENABLED && INTAKE_ENABLED && APRILTAGS_ENABLED && DRIVEBASE_ENABLED
```

Comment:
Must all of these be enabled for visionLaunch?  Do you require LAUNCHER AND INTAKE or is it at least one of them so an OR?